### PR TITLE
feat: tiktok_offline_events added support for all Standard  events

### PR DESCRIPTION
**Pull request by anantjain45823**
## What are the changes introduced in this PR?
Adding support for all Standard event supported by tiktok_ads_offline. [Tiktok Docs](https://ads.tiktok.com/help/article/about-offline-conversions?lang=en)
Write a brief explainer on your code changes.
Currently we support only 4 standard events out of 14 standard events that tiktok supports.
in this PR I have added support for the remaining 10 events ones as well.

## What is the related Linear task?

Resolves INT-1554

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [x] My code follows the style guidelines of this project

- [x] **No breaking changes are being introduced.**

- [x] All related docs linked with the PR?

- [x] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [x] Is the PR limited to 10 file changes?

- [x] Is the PR limited to one linear task?

- [x] Are relevant unit and component test-cases added?

### Reviewer checklist

- [x] Is the type of change in the PR title appropriate as per the changes?

- [x] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3094/test-report.html

File: src/v0/destinations/tiktok_ads_offline_events/config.js [hunk 1]
```diff
@@ -23,8 +23,17 @@ const EVENT_NAME_MAPPING = {
   contact: 'Contact',
   submitform: 'SubmitForm',
   subscribe: 'Subscribe',
+  'product added to wishlist': 'AddToWishlist',
```
Comment (koladilip) [inline src/v0/destinations/tiktok_ads_offline_events/config.js:26]: better to follow alphabetical order for keys

**Review by shrouti1507 — APPROVED** [review]

File: src/v0/destinations/tiktok_ads_offline_events/config.js (hunk 1 above)
Comment (anantjain45823) [reply src/v0/destinations/tiktok_ads_offline_events/config.js]: Done

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3094/test-report.html

File: src/v0/destinations/tiktok_ads_offline_events/config.js [hunk 2]
```diff
@@ -20,11 +20,20 @@ const PARTNER_NAME = 'RudderStack';
...
+  'product added': 'AddToCart',
+  clickbutton: 'ClickButton',
+  completeregistration: 'CompleteRegistration',
   contact: 'Contact',
+  download: 'Download',
+  search: 'Search',
   submitform: 'SubmitForm',
   subscribe: 'Subscribe',
+  viewcontent: 'ViewContent',
 };
```
Comment (koladilip) [inline src/v0/destinations/tiktok_ads_offline_events/config.js:36]:
```suggestion
const EVENT_NAME_MAPPING = {
  'addpaymentinfo': 'AddPaymentInfo',
  'addtocart': 'AddToCart',
  'addtowishlist': 'AddToWishlist',
  'checkout started': 'InitiateCheckout',
  'checkout step completed': 'CompletePayment',
  'clickbutton': 'ClickButton',
  'completeregistration': 'CompleteRegistration',
  'contact': 'Contact',
  'download': 'Download',
  'order completed': 'PlaceAnOrder',
  'payment info entered': 'AddPaymentInfo',
  'product added': 'AddToCart',
  'product added to wishlist': 'AddToWishlist',
  'search': 'Search',
  'submitform': 'SubmitForm',
  'subscribe': 'Subscribe',
  'viewcontent': 'ViewContent',
};
```

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3094/test-report.html

**Review by koladilip — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Pull request merged**